enclave-runner/src/library.rs

```diff
@@ -68,13 +71,13 @@ impl Library {
     /// The caller must ensure that the parameters passed-in match what the
     /// enclave is expecting.
     pub unsafe fn call(
-        &self,
+        &mut self,
```

Comment:
This is a public API change, so we need a minor version bump

Reply:
Are there docs somewhere (maybe the EDP website?) that also need to be updated?

Reply:
This API change is not appropriate. Multiple threads should be able to call into the enclave.